.gitlab/ci/on-push.yml

```diff
@@ -7,6 +7,173 @@
   - echo "======= Download configuration ========"
   - ./Tests/scripts/download_demisto_conf.sh  >> $ARTIFACTS_FOLDER/logs/installations.log
 
+.notify-slack:
+  after_script:
+    - echo "=====Notify Slack====="
+    - |
+      if [ -n "${SLACK_ONLY_IF}" ]; then
+        ./Tests/scripts/slack_notifier.sh "$SLACK_TEST_TYPE" "$SLACK_ENV_RESULTS" "$SLACK_CHANNEL"
+      else
+        echo "Skipping Slack"
+      fi
+
+run-unittests-and-lint:
+  needs: []
+  stage: unittests-and-validations
+  rules:
+    - if: '$FORCE_BUCKET_UPLOAD != "true"'
```

Comment:
It needs to run in every build except the force upload and instance testing, that includes:
- commit
- nightly
- sdk-nightly
- upload